src/templates/app/user/security/limits.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Table } from '../../../_common/components/elements.jsx';
+import Loading from '../../../_common/components/loading.jsx';
```

Comment:
are we still using rules from spacing in binary static 😅

Reply:
@mustofa-binary we align `=`, not `from` as far as I remember :)